app/controllers/api/account.php

```diff
@@ -1622,17 +1622,18 @@
     ->label('sdk.offline.model', '/account')
     ->label('sdk.offline.key', 'current')
     ->param('email', '', new Email(), 'User email.')
-    ->param('password', '', new Password(), 'User password. Must be at least 8 chars.')
+    ->param('password', '', new Password(), 'User password. Must be at least 8 chars.', true)
```

Comment:
Let's add that this is not required for users that haven't set the password, but will throw exception otherwise.